motor.h

```diff
@@ -1,7 +1,10 @@
 #ifndef MOTOR_H
 #define MOTOR_H
 
+
```

Comment:
I didn't mean here. I mean to separate the `#include` from the `#define` below it, not above it. That is:
```cpp
#include "Arduino.h"


#define MOTOR_FORWARD 0
#define MOTOR_BACKWARD 1
```